lib/auth0/api/v2/resource_servers.rb

```diff
@@ -62,6 +62,17 @@ def delete_resource_server(resource_server_id)
           delete(path)
         end
 
+        # Updates a resource server.
+        # @see https://auth0.com/docs/api/management/v2#!/Resource_Servers/patch_resource_servers_by_id
+        # @param id [string] The id or audience of the resource server to update.
+        # @param options [hash] The Hash options used to define the resource servers's properties.
```

Comment:
Let's change this parameter name to `body`. We typically use `options` for URL parameters.

Reply:
Done.